cellprofiler/modules/identifyobjectsmanually.py

```diff
@@ -27,7 +37,6 @@
 ============ ============ ===============
 YES          NO           NO
 ============ ============ ===============
-
```

Comment:
There should be a "See also" section header with the `HELP_ON_SAVING_OUTLINES` text here.